linebot/message.go

```diff
@@ -83,6 +86,12 @@ func (m *TextMessage) WithSender(sender *Sender) SendingMessage {
 	return m
 }
 
+// AddEmoji method of TextMessage
```

Comment:
maybe need to add this method to `SendingMessage` interface?

Reply:
@nasa9084 -san,
The emoji value only works for TextMessage, and not working for ImageMessage or other message types.

If I add it as an interface for XendMessage, I might do nothing if message type is not TextMessage.

How do you think?

e.g. 

```
// AddEmoji method of ImageMessage
func (m *ImageMessage) AddEmoji(emoji *Emoji) SendingMessage {
	return m
}
```

Reply:
I'm newer person for this project so I'm not sure though, I think the method is considered to be used as method chain like `m.AddEmoji(emoji1).AddEmoji(emoji2)`.

I can see:

1. `AddEmoji` is returning `SendingMessage` interface
2. the field `emojis` is private field
3. `NewTextMessage` does not have argument to set emojis

so users cannot set multiple emojis without type assertion, if `SendingMessage` does not have `AddEmoji` method.

I think we have some options:

* Add `AddEmoji` method to `SendingMessage` interface
* Change not to return `SendingMessage` for all methods, means remove `SendingMessage` interface
* Change `AddEmoji` method to `AddEmojis` to be able to pass `[]*Emoji`
* Change the `NewTextMessage` to be able to pass emojis
* ... or maybe other choice

HDYT?

cc/ @oklahomer @k2wanko

Reply:
@nasa9084 @oklahomer  Any conclusion?

Reply:
@nasa9084 Could you check again? I have fixed your question.

Reply:
sorry to be late reply, please wait for a while....

Reply:
Thank you, I checked and LGTM